raiden/ui/cli.py

```diff
@@ -223,6 +223,11 @@ def convert(self, value, param, ctx):
         ),
         default=True,
     ),
+    click.option(
+        '--eth-client-communication',
+        help='Print all communication with the underlying eth client',
```

Comment:
I believe a simple flag would be better, see the `--log-json` option for the format.

Reply:
oh yes indeed, since it's a boolean it should be an include/not include flag. Thanks for the heads up.